crates/biome_glob/src/lib.rs

```diff
@@ -1,31 +1,86 @@
-use biome_rowan::{TextRange, TextSize};
+//! biome_glob provides a glob and glob list with exceptions matching.
+//!
+//! ## Matching a path against a glob
+//!
+//! You can create a glob from a string using [core::str::FromStr::from_str] or the corresponding method `parse`.
+//! A glob can match against anything that can be turned into a [std::path::Path].
+//! This is for example the aqcse of strings.
```

Comment:
```suggestion
//! You can create a glob from a string using [core::str::FromStr::from_str] or the corresponding method `parse`.
//! A glob can match against anything that can be turned into a [std::path::Path].
//! This is, for example, the access of strings.
```

I'm not really sure what `This is for example the aqcse of strings.` means 😅